universal-application-tool-0.0.1/app/services/question/ReadOnlyQuestionService.java

```diff
@@ -27,6 +29,12 @@
   /** Returns all repeater question definitions. */
   ImmutableList<RepeaterQuestionDefinition> getUpToDateRepeaterQuestions();
 
+  /** Returns all scalars for this version. */
+  ImmutableMap<Path, ScalarType> getAllScalars();
```

Comment:
Nit (blocking): this might be from a merge conflict. It was deleted in a recent PR.

Reply:
Ah thanks - I saw changes that use it but I see now that I ... need to remove them?  In progress...